### DISCUSSION TITLE
Tablet stops working in windows after opening osu!

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23793] [Q&A] [answered]
### Type

Other

### Bug description

Once I open osu!, The windows no longer detects my tablet's input.
(This problem fixes with OTD)

### Version

2023.605.0-lazer (not this version only, I had it earlier too)

### Logs

[runtime.log](https://github.com/ppy/osu/files/11666646/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11666647/updater.log)
[database.log](https://github.com/ppy/osu/files/11666648/database.log)
[network.log](https://github.com/ppy/osu/files/11666651/network.log)
[performance.log](https://github.com/ppy/osu/files/11666652/performance.log)
[input.log](https://github.com/ppy/osu/files/11666649/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11666650/legacy-ipc.log)

**Comment by peppy** [comment]
Need far more detail than what you'd provided.

What "doesn't detect"? What tablet? Does it resolve when you close osu!? Can you show it in a video?

**Reply by rvnexe** [reply]
"doesn't detect" = My tablet input no longer detects by windows. I mean, when I move my pen on the tablet, my cursor will not move.
My tablet is Star G640S
And problem will not resolve after closing osu. I'll record a video shortly

**Reply by peppy** [reply, marked as answer]
This may be something you need to report over at https://github.com/OpenTabletDriver/OpenTabletDriver. I don't think we do anything at our end that should cause a device to stop working permanently, so it's either OTD or your drivers at fault.

**Comment by rvnexe** [comment]
https://github.com/ppy/osu/assets/89677246/925cde1e-819c-4b76-8f1b-6d532f2847d1

**Reply by rvnexe** [reply]
(I think there's a problem with video's encoding. It's blank in web view. Open it with VLC or smth.)
